pennylane_lightning/core/src/simulators/lightning_qubit/StateVectorLQubitRaw.hpp

```diff
@@ -99,6 +100,15 @@ class StateVectorLQubitRaw final
      */
     auto getData() -> ComplexT * { return data_; }
 
+    /**
+     * @brief Get a copy of underlying data.
+     *
+     * @return a std::vector<ComplexT> object that stores statevector data.
+     */
+    auto getDataVector() -> std::vector<ComplexT> {
+        return std::vector<ComplexT>{getData(), getData() + getLength()};
```

Comment:
```suggestion
        return std::vector<ComplexT>{data_, data_+ length_};
```

No need for the function calls here, since the private data is accessible.

Reply:
Done!